examples/pytorch/gcn/README.md

```diff
@@ -2,33 +2,29 @@ Graph Convolutional Networks (GCN)
 ============
 
 - Paper link: [https://arxiv.org/abs/1609.02907](https://arxiv.org/abs/1609.02907)
-- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original code is 
-implemented with Tensorflow for the paper. 
+- Author's code repo: [https://github.com/tkipf/gcn](https://github.com/tkipf/gcn). Note that the original implementation is based on Tensorflow.
 
-Dependencies
-------------
-- PyTorch 0.4.1+
-- requests
-
-``bash
-pip install torch requests
-``
-
-Codes
+Available examples
 -----
-The folder contains three implementations of GCN:
-- `gcn.py` uses DGL's predefined graph convolution module.
-- `gcn_mp.py` uses user-defined message and reduce functions.
-Modify `train.py` to switch between different implementations.
+The folder includes an implementations of GCN using DGL intrinsic graph convolution module.
 
-Results
+How to run
 -------
 
-Run with following (available dataset: "cora", "citeseer", "pubmed")
+### DGL intrinsic GCN module
```

Comment:
We typically use the term "built-in" rather than "intrinsic".

Reply:
Got it. I will make the replacements